### PR TITLE
Feature/apn 5 matching

### PR DESCRIPTION
**Pull request by MichaelSchule**

### CODE REVIEW

File: ApollonBackend/ApollonBackend/Authentication/ClientCredentialsAuthenticator.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    public class ClientCredentialsAuthenticator : IAuthenticationProvider
+    {
+        private readonly IHttpClientFactory m_httpClientFactory;
+
+        public ClientCredentialsAuthenticator(IHttpClientFactory httpClientFactory)
+        {
+            m_httpClientFactory = httpClientFactory;
+        }
+
+        private const string c_clientId = "dbc613297d2546de89b52731fa42b508";
```
Comment (pg-co) [inline ApollonBackend/ApollonBackend/Authentication/ClientCredentialsAuthenticator.cs:20]: Create Environtment variable

Comment (pg-co) [inline ApollonBackend/ApollonBackend/Authentication/ClientCredentialsAuthenticator.cs:20]: Replace with Environment Variable

**Review by pg-co — CHANGES_REQUESTED** [review]
LGTM

File: ApollonBackend/ApollonBackend/Authentication/ClientCredentialsAuthenticator.cs (hunk 1 above)
Comment (MichaelSchule) [reply ApollonBackend/ApollonBackend/Authentication/ClientCredentialsAuthenticator.cs:20]: private const only for test purposes, will later change to be in a provider class

**Pull request merged**